### PR TITLE
feat(curriculum): add relational databases cert as upcoming change

### PR DESCRIPTION
**Pull request by moT01**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #XXXXX

ToDo:
- [ ] Reorder superBlocks to the proper order 
- [ ] Find and add an icon for relational databases cert
- [ ] Remove unneeded commented out code
- [x] Remove CSS file I added to try and fix something that didn't work
- [ ] Find a way to remove the vertical scroll bar that shows up from the line-height added by bootstrap 
- [ ] Transfer all tutorials to fCC organization and update URL's in challenge md files
- [ ] Finish creating the tutorials
- [ ] Fix tests
- [x] Rename all Strove references to CodeAlly
- [ ] Add tests? If possible
- [ ] Write intros for placeholders
- [ ] Add cert back to settings page

Workaround for testing individual tutorials if you don't want to pull down this PR. Create an HTML file locally and add this to it:
```
<html>
  <body style="margin: 0; padding: 0; overflow: hidden;">
    <iframe
      sandbox='allow-modals allow-forms allow-popups allow-scripts allow-same-origin'
      src='http://codeally.io/embed/?repoUrl=https://github.com/moT01/.learn-bash-scripting-by-building-five-programs'
      style="width: 100%; height: 100%;"
      title='Editor'
    />
  </body>
</html>
```
Replace the URL with the project you want to test. You may not have permissions to run these, feel free to ask if you are unsure or want them.

### CODE REVIEW

**Comment by moT01** [comment]
Alright, I think we can try and get this merged in so I don't have to keep rebasing. I suppose the main things to check would be that this cert doesn't show up anywhere with `SHOW_UPCOMING_CHANGES=false`, and that none of the existing curriculum is broken. Feel free to make any suggestions.

**Review by raisedadead — APPROVED** [review]

File: curriculum/schema/challengeSchema.js [hunk 1]
```diff
@@ -44,7 +44,12 @@ const schema = Joi.object()
       indexjsx: fileJoi
     }),
     guideUrl: Joi.string().uri({ scheme: 'https' }),
-    helpCategory: Joi.valid('JavaScript', 'HTML-CSS', 'Python'),
+    helpCategory: Joi.valid(
+      'JavaScript',
+      'HTML-CSS',
+      'Python',
+      'Relational Databases'
```
Comment (ShaunSHamilton) [inline curriculum/schema/challengeSchema.js:51]: Silly question: Is the helpCategory allowed to have whitespace?

**Review by ShaunSHamilton — APPROVED** [review]
This LGTM 👍 - does not break current curriculum, but does not open Relational DB lessons for me

File: curriculum/schema/challengeSchema.js (hunk 1 above)
Comment (moT01) [reply curriculum/schema/challengeSchema.js:51]: Not sure - I wondered that myself.

**Pull request merged**